### PR TITLE
posts: Login issues blog post

### PR DESCRIPTION
**Pull request by garrett**
Please review ASAP. We need this merged and live on the site quickly.

### CODE REVIEW

**Comment by garrett** [comment]
It's rough in some areas, especially with regard to the lack of RHEL status and workarounds. I included my simple sed 1-liner as a fallback (which worked for me and one of our users), but urge people to use packages instead.

File: _posts/2022-10-20-login-issues.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+In most cases, update. [Cockpit 277](cockpit-277), released last month, has the fix.
+
+Updating Cockpit should work for Arch, CentOS Stream, Debian, Fedora, openSUSE, non-LTS versions of Ubuntu.
+
+If updating doesn't work (in Ubuntu LTS some versions of RHEL, Alma Linux, and Rocky Linux), then hopefully it will work very soon. For the time being, please [look at the bottom of this post for some workarounds](#workarounds).
+
+## Summary
+
+Logging in with new versions of browsers into a few older versions of Cockpit is currently broken due to a check in the login code. This check used to pass in all browsers, but very recently every browser have changed the way they parse the string. Firefox release 106 — released this week — has this parsing change; all other browsers will have it in upcoming versions too.
```
Comment (jelly) [inline _posts/2022-10-20-login-issues.md:19]:
```suggestion
Logging in with new versions of browsers into a few older versions of Cockpit is currently broken due to a check in the login code. This check used to pass in all browsers, but very recently every browser has changed the way they parse the string. Firefox release 106 — released this week — has this parsing change; all other browsers will have it in upcoming versions too.
```

File: _posts/2022-10-20-login-issues.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+We fixed this bug as soon as it was discovered (a month ago) and pushed [a new release of Cockpit](cockpit-277) and also backported it as a hotfix to all the "stable" slower-moving distributions.
+
+All distributions have had the fix for a around a month. Most distributions have accepted and published the fix, including Arch Linux, CentOS Stream 8 & 9, Debian, Fedora, openSUSE Tumbleweed, Red Hat Linux Enterprise Linux versions prior to 8.8, and Ubuntu Kenetic.
+
+Unfortunately, some of the slower-moving distributions ("enterprise" and "long term support") haven't updated yet — and this notably includes Red Hat Enterprise Linux 8.6 & 9.0, Ubuntu 22.04 LTS, and Rocky Linux.
```
Comment (jelly) [inline _posts/2022-10-20-login-issues.md:25]: Isn't this a bit over verbose? Maybe just say: "Due to the nature of enterprise/LTS distributions the fixes have not landed yet in X, Y,Z .

File: _posts/2022-10-20-login-issues.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+Firefox ESR (Extended Support Release) is slower-moving and is on 102.x and is not affected, nor will it be affected until Q3 2023 (2023-07-04 specifically, according to [the Firefox release schedule](https://wiki.mozilla.org/Release_Management/Calendar)). 
+
+## Workarounds
+
+### Ubuntu
+
+#### Official packages in "proposed"
+
+For Ubuntu, the fix should finally be out in the main repo next week.
```
Comment (jelly) [inline _posts/2022-10-20-login-issues.md:39]:
```suggestion
For Ubuntu, the fix should be out in the main repo next week.
```

I think it's better to word it more neutral.

File: _posts/2022-10-20-login-issues.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+In the meantime, the fixed version of Cockpit has just recently been accepted to the proposed repository (a staging repo). You can enable this repository or even just cherry-pick specific packages (such as Cockpit packages, like `cockpit-ws`). More information is available at [Ubuntu's EnableProposed wiki page](https://wiki.ubuntu.com/Testing/EnableProposed) and [at Launchpad](https://bugs.launchpad.net/ubuntu/+source/cockpit/+bug/1990623/comments/10). If you test this and it works for you (it should!) please also leave a comment [at the Launchpad issue](https://bugs.launchpad.net/ubuntu/+source/cockpit/+bug/1990623) to let them know to push it to the main repos sooner, so everyone will get the fix!
```
Comment (jelly) [inline _posts/2022-10-20-login-issues.md:41]:
```suggestion
In the meantime, the fixed version of Cockpit has been accepted to the proposed repository (a staging repo). You can enable this repository or even just cherry-pick specific packages (such as Cockpit packages, like `cockpit-ws`). More information is available at [Ubuntu's EnableProposed wiki page](https://wiki.ubuntu.com/Testing/EnableProposed) and [at Launchpad](https://bugs.launchpad.net/ubuntu/+source/cockpit/+bug/1990623/comments/10). If you test this and it works for you (it should!) please also leave a comment [at the Launchpad issue](https://bugs.launchpad.net/ubuntu/+source/cockpit/+bug/1990623) to let them know to push it to the main repos sooner, so everyone will get the fix!
```

File: _posts/2022-10-20-login-issues.md (hunk 2 above)
Comment (garrett) [reply _posts/2022-10-20-login-issues.md:25]:
```suggestion
Due to the nature of enterprise/LTS distributions the fixes have not landed yet in Red Hat Enterprise Linux 8.6 & 9.0, Ubuntu 22.04 LTS, Alma Linux, and Rocky Linux.
```

**Comment by martinpitt** [comment]
Many thanks @garrett for the great write-up! :+1: Aside from the "Kenetic" spelling error it looks correct to me!

**Pull request merged**